app/test/test_audio_similarity.py

```diff
@@ -152,17 +152,17 @@ def test_search_by_doc_id(self):
         db.session.commit()
         with patch('app.main.lib.shared_models.audio_model.AudioModel.get_by_doc_id_or_url', ) as mock_get_by_doc_id_or_url:
```

Comment:
isn't the function being mocked here one of the ones that is being tested?

Reply:
Partially. The goal of this test is not to check whether or not the getting of an individual item works or not, the goal is to make sure that that *rest* of the *search* functionality works, *conditional* on it being a lookup via doc id. If you'd like, I can relax that constraint and rewrite this test.